ui/components/app/qr-hardware-popover/qr-hardware-wallet-importer/qr-hardware-wallet-importer.component.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  submitQRHardwareCryptoAccount,
+  submitQRHardwareCryptoHDKey,
+} from '../../../../store/actions';
+import BaseReader from '../base-reader';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const QRHardwareWalletImporter = ({ handleCancel, setErrorTitle }) => {
+  const t = useI18nContext();
+  const handleSuccess = (ur) => {
```

Comment:
better to make this an async function?

Reply:
yes, will update

Reply:
update please

Reply:
done

Reply:
this was not turned into an async function